airbyte-integrations/bases/debezium/src/main/java/io/airbyte/integrations/debezium/internals/AirbyteSchemaHistoryStorage.java

```diff
@@ -127,8 +218,9 @@ public static AirbyteSchemaHistoryStorage initializeDBHistory(final Optional<Jso
     }
     final Path dbHistoryFilePath = dbHistoryWorkingDir.resolve("dbhistory.dat");
 
-    final AirbyteSchemaHistoryStorage schemaHistoryManager = new AirbyteSchemaHistoryStorage(dbHistoryFilePath);
-    schemaHistoryManager.persist(schemaHistory);
+    final AirbyteSchemaHistoryStorage schemaHistoryManager =
+        new AirbyteSchemaHistoryStorage(dbHistoryFilePath, schemaHistoryInfo.compressionEnabled());
```

Comment:
nit: since `compressionEnabled` is a property of a connector and not of the connection, can we dis-associate this flag with schemaHistoryInfo?